src/Orchard.Cms.Web/Modules/Orchard.Settings/Views/Admin/RestartSite.cshtml

```diff
@@ -0,0 +1,8 @@
+﻿<div class="alert alert-warning">
+    <h4>@T["Site Restart"]</h4>
```

Comment:
Should this be visible to a user who has not got permission to restart the server?  

Should this be wrapped in something like 

```c#
@if user.haspermission(Permissions.Restart){
<this block>
}
```